build.gradle.kts

```diff
@@ -16,11 +16,7 @@ repositories {
 kotlin {
     targets {
         jvm {
-            compilations.all {
-                kotlinOptions {
-                    jvmTarget = "1.8"
```

Comment:
Do folks usually omit the jvmTarget to mean 1.6, or would it be better to be explicit?

Reply:
It will use the Kotlin default, which is exactly what we want here.

Reply:
Ah 👍